diff --git a/files/handler-393b2283.js b/files/handler-393b2283.js
index c5588bfa8e4eb54fed732724e95aa6dab677a944..c6a50d6de94641f8e7f71eddaf704b48017d190c 100644
--- a/files/handler-393b2283.js
+++ b/files/handler-393b2283.js
@@ -1052,7 +1052,14 @@ const address_header = env('ADDRESS_HEADER', '').toLowerCase();
 const protocol_header = env('PROTOCOL_HEADER', '').toLowerCase();
 const host_header = env('HOST_HEADER', 'host').toLowerCase();
 
-const __dirname = path.dirname(fileURLToPath(import.meta.url));
+let __dirname;
+try {
+  __dirname = path.dirname(fileURLToPath(import.meta.url))
+} catch (e) {
+  try {
+    __dirname = path.dirname(resolve(".", process.argv[1] ))
+  } catch (e) { }
+}
 
 /**
  * @param {string} path